features/_your_results/salary_below_minimum_threshold.feature

```diff
@@ -0,0 +1,38 @@
+Feature: Employer contributions do not increase when user's salary is less than the minimum threshold
+  In order to understand that there is no requirement for employer contributions to increase over time
+  As an employee earning under £5876
```

Comment:
From reading the story [here](https://moneyadviceservice.tpondemand.com/entity/8565) my guess is that this figure of £5,876 is based on annual salary? I think the description needs to be clearer. This will help me dry-trace the cuke scenarios. I will continue the review once this blocker has been cleared up

Reply:
@Tr4pSt3R  The salary is £5,876 annually. I will update the description, however I disagree that this is a blocker for you to complete your review. Whilst I understand the desire for clarity and not making assumptions, in this instance, I believe common sense can be exercised - the employer contributions are not going to be different for 5,876 per year, per month, per 4-weeks and per week.